app/js/views/services.jsx

```diff
@@ -69,7 +69,3 @@ export default class Services extends BaseView {
     );
   }
 }
-
-Services.propTypes = {
```

Comment:
> If I understand correctly, it doesn't impact us right now.

It impacts, but it's very-very-very minor thing (just 3 noop proptypes are left inside minified file) :)